src/compas_viewer/viewer.py

```diff
@@ -37,6 +37,17 @@ def __init__(self, config: Config = None, **kwargs):
         self.controller = Controller(self.config)
 
         self.ui = UI()
+        self._scene = None
+
+    @property
+    def scene(self) -> ViewerScene:
+        if self._scene is None:
+            self._scene = ViewerScene()
+        return self._scene
+
+    @scene.setter
+    def scene(self, scene: Scene):
+        self._scene = ViewerScene.__from_data__(scene.__data__)
```

Comment:
Not really ideal....But this can be potentially avoided if we remove the layer of ViewerScene completely.